src/api/meeting.gql

```diff
@@ -40,6 +40,21 @@ subscription circleMeetings($circleId: uuid!) {
   }
 }
 
+
+query meetingsAtSameTime($NMstartDate: timestamptz!, $NMendDate: timestamptz!) {
+  meeting(where: { archived: {_eq: false}, 
+  _or: [
+    { startDate: { _gt: $NMstartDate }, endDate: { _lt: $NMendDate } }, 
+    { startDate: { _lte: $NMstartDate }, endDate: { _gte: $NMendDate } },
+    {_and: [{ startDate: { _lte: $NMstartDate }, endDate: { _gte: $NMstartDate } }, { endDate: { _lte: $NMendDate } }]},
+    {_and: [{ startDate: { _gte: $NMstartDate }, endDate: { _gte: $NMendDate } }, { startDate: { _lt: $NMendDate } }]}
```

Comment:
Tu dois pouvoir faire plus simple et performant avec la fonction postgres  `OVERLAPS`.
https://database.guide/how-to-test-for-overlapping-dates-in-postgresql/

Je ne crois pas qu'Hasura supporte nativement cette fonction, donc tu peux créer une fonction Hasura `getOverlappingMeetings` :
https://hasura.io/docs/latest/schema/postgres/custom-functions/